titan-server/src/config.rs

```diff
@@ -1,13 +1,15 @@
 use rocket::Config;
 use rocket::config::ConfigError;
 
-pub struct AuthConfig {
-    pub secret_key: String
+pub struct AppConfig {
```

Comment:
Renamed `AuthConfig` to `AppConfig`. This struct is used to access config constants. Creating a new struct for each config option is a waste of space.

Reply:
I like this rename. I think this structure works well for now. :+1: